.github/workflows/publish-npm-package.yml

```diff
@@ -0,0 +1,28 @@
+name: 🏷 Bump Version & 📦 Publish Package to npm
+jobs:
```

Comment:
We should have a 

`on: workflow_dispatch`

to be able to trigger it manually

Reply:
@Delawen can't you just go to the Actions tab and click "Run workflow"? where would you want to trigger it from?